web/src/App.tsx

```diff
@@ -73,6 +75,7 @@ export default function App(): ReactElement {
       >
         <title>{`Electricity Maps | ${t('misc.maintitle')}`}</title>
         <meta property="og:locale" content={i18n.languages[0]} />
+        {canonicalLink}
```

Comment:
@cadeban would you prefer if this was a component instead?
Might be better DX to so it better matches how it used?

Reply:
Yeah, that is a concern. Not sure what is the best way to do it as I don't want to duplicate the functionality across the pages/components.

Reply:
Switched it up to use a combination of a hook + link elements which should adress both concerns.

Reply:
Yeah, I hear you re: duplication. It seems like the `react-helmet-async` [unions](https://github.com/nfl/react-helmet?tab=readme-ov-file#features) all of the Helmet definitions. Maybe we could do something like define a `CanonicalHelmet` component that could consolidate the hook, link, and helmet and then update `app.tsx` to something like below?

```jsx
// app.tsx
...
  <Helmet htmlAttributes={{...}} prioritizeSeoTags>
    <title>{`Electricity Maps | ${t('misc.maintitle')}`}</title>
    <meta property="og:locale" content={i18n.languages[0]} />
  </Helmet>
  <CanonicalHelmet />
...
```

Reply:
(Otherwise, lgtm!)

Reply:
We will be putting more stuff in the other helmets like title and description manipulation so I'm not sure having multiple helmets on each page would be good either.

Perhaps we can go with this for now and then look for optimisations once we have all the tags in place?

Reply:
sgtm!